tests/phpunit/tests/media.php

```diff
@@ -4118,6 +4213,9 @@ public function test_the_excerpt_does_not_affect_content_media_count() {
 	 * that featured image not being lazy-loaded, since the images in the post content aren't displayed in the excerpt.
 	 *
 	 * @ticket 56588
+	 * @ticket 58235
+	 *
+	 * @since 6.3.0 Apply fetchpriority="high" on the LCP image.
```

Comment:
Please avoid `@since` annotations in tests, since the underlying change is not happening here.

Reply:
Removed.